src/main/java/com/cloudbees/jenkins/support/impl/LoadStats.java

```diff
@@ -314,16 +313,4 @@ public boolean shouldBeFiltered() {
             return false;
         }
     }
-
-    /**
-     * Converts "aCamelCaseString" into "A sentence case string".
-     *
-     * @param camelCase camelCase.
-     * @return Sentence case.
-     */
-    private static String camelCaseToSentenceCase(String camelCase) {
```

Comment:
Can we keep the camel case to sentence logic ? 
https://github.com/jenkinsci/jenkins/blob/24a2e7c78278e591706a572896c0cabd69f4d705/core/src/main/resources/hudson/model/Messages.properties#L374-L381.